src/Elastic.Apm/Filters/ErrorContextSanitizerFilter.cs

```diff
@@ -22,7 +22,7 @@ public IError Filter(IError error)
 			{
 				if (realError.Context.Request?.Headers != null && realError.ConfigSnapshot != null)
 				{
-					foreach (var key in realError.Context?.Request?.Headers?.Keys)
+					foreach (var key in realError.Context?.Request?.Headers?.Keys.ToList())
```

Comment:
We need `.ToList()` here, otherwise we can't change the dictionary within the `foreach`. Same in `HeaderDictionarySanitizerFilter.cs `